config/armada/config.yaml

```diff
@@ -17,60 +17,99 @@ grpc:
     permitWithoutStream: false
 redis:
   addrs:
-    - "localhost:6379"
+    - "redis:6379"
   password: ""
   db: 0
   poolSize: 1000
 eventsRedis:
   addrs:
-    - "localhost:6379"
+    - "redis:6379"
   password: ""
   db: 0
   poolSize: 1000
 eventsApiRedis:
   addrs:
-    - "localhost:6379"
+    - "redis:6379"
   password: ""
   db: 1
   poolSize: 1000
 defaultToLegacyEvents: true
 scheduling:
   preemption:
-    # if enabled, Armada will accept submitted jobs which have a priorityClassName field
-    # otherwise it will refuse the jobs on submit
-    enabled: false
-    # priorityClasses field is a map of priority class names to priority values
-    # defined priority classes should exist with the same priority in all executor clusters
-    priorityClasses: {}
-    # if preemption is enabled, Armada server will set the priorityClassName in the job's podSpec
-    # with the value of defaultPriorityClass, if the podSpec does not already reference a priority class
+    enabled: true
+    priorityClasses:
+      armada-default:
+        priority: 1000
+        maximalResourceFractionPerQueue:
+          memory: 0.99
+          cpu: 0.99
+      "": # Treat missing PC the same as armada-default.
+        priority: 1000
+        maximalResourceFractionPerQueue:
+          memory: 0.99
+          cpu: 0.99
+      armada-preemptible:
+        priority: 900
+        maximalResourceFractionPerQueue:
+          memory: 0.99
+          cpu: 0.99
+    # Default to empty PC to not require PCs be set up in worker clusters.
     defaultPriorityClass: ""
-  useProbabilisticSchedulingForAllResources: true
-  queueLeaseBatchSize: 200
-  maximumLeasePayloadSizeBytes: 7340032 # 1024 * 1024 * 7
+  queueLeaseBatchSize: 1000
   minimumResourceToSchedule:
-    memory: 100000000 # 100Mb
-    cpu: 0.25
+    memory: 1000000
```

Comment:
let's keep the comment to say what this is as it's not obvious what the units are

Reply:
Fixed